docs/search relevance/test_3.md

```diff
@@ -0,0 +1,22 @@
+# Test 3
```

Comment:
Nit: Could we potentially title this - and name them following other docs i.e
`0003-adding_notes.md`